package/src/common/Input/InputCryptoAddr.tsx

```diff
@@ -59,7 +68,15 @@ const InputCryptoAddr = React.forwardRef<HTMLDivElement, InputCryptoAddrType>(
       },
       [handleInputChange, collected.cryptocurrencyAddress]
     );
-
+    const walletFieldClick = useCallback(async () =>{
+      const gtmData = {
+        event: GtmEvent.ELEMENT_CLICK,
+        action: GtmEventAction.WALLET_FORM,
+        category: GtmEventCategory.FIELD,
+        label: GtmEventLabel.WALLET_ADDRESS,
+      };
+      sendDataToGTM(gtmData);
+    },[]);
```

Comment:
1. Lint warning
2. This method can be removed from here and put in BodyFormView. Use the single method to send events same as other scenarios.
3. async is not required here. No asynchronous tasks are happening here